docs/install/docker.md

```diff
@@ -1,46 +1,59 @@
 # Docker Installation
 
+- [Note about Windows and Git configuration](#note-about-windows-and-git-configuration)
 - [First Time Installation](#first-time-installation)
-- [Erase and Start Fresh](#erase-and-start-fresh)
-- [Overriding URL and Ports](#overriding-url-and-ports)
-- [Reviewing Code](#reviewing-code)
 - [Logs](#logs)
+- [Development](#development)
+  - [Erase and Start Fresh](#erase-and-start-fresh)
+  - [Reviewing Code](#reviewing-code)
+
 
 If you are already set up with Docker, you may wish to refer to the
 [Docker Command Quick Reference](/docs/advanced/docker-reference.md) guide.
 
-## First Time Installation
-
-### IMPORTANT INFORMATION FOR WINDOWS USERS
+## Note about Windows and Git configuration
 If you have git setup to use `CRLF` but commit `LF` (this is the default) and you are going to use docker-desktop you will run into
 the problem that the start scripts no longer work. You will see error messages like `web_1    | /bin/sh: /entrypoint.sh: /bin/bash^M: bad interpreter: No such file or directory`. To fix, you need to force Git to use `LF` in your working directory by running `git config core.autocrlf false`
 This will only affect this repository (add `--global` if you want it to be the default for all your Git repositories).
 
+## First Time Installation
 Install Docker using [instructions on Docker's website](https://www.docker.com/products/docker-desktop).
 
-For typical development, clone the `dev` branch, then copy and modify the
-
-overrides:
+For deploy to a production environment, check out the `master` branch by running
+```bash
+git clone https://github.com/open-oni/open-oni.git
+git checkout master
```

Comment:
> For deploy**ing** to a production environment
??

Could you change `master` to `main` as well. I've made `main` for the repo and am merging our next release into it rather than `master` :smile: